packages/devextreme/js/__internal/ui/list/m_list.base.ts

```diff
@@ -833,8 +862,9 @@ export class ListBase extends CollectionWidget<ListBaseProperties, Item> {
   }
 
   _processGroupCollapse(e: DxEvent<MouseEvent | PointerEvent | TouchEvent | KeyboardEvent>): void {
```

Comment:
after parallel PR merged
```suggestion
  _processGroupCollapse(e: InteractionEvent): void {
```